cli/bpmetadata/repo.go

```diff
@@ -74,3 +80,8 @@ func getBpSubmoduleName(bpPath string) string {
 
 	return ""
 }
+
+func convertTitleCaseToKebabCase(title string) string {
```

Comment:
We can use the strcase pkg which we use in the other subcommands too
https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/f67d36cf137f826535e8737af4849631f8d504f2/cli/bptest/convert.go#L167